stacks/readme.txt

```diff
@@ -0,0 +1,40 @@
+=== Stacks ===
+Contributors: @SaxonF @scruffian
```

Comment:
According to [the guidelines](https://make.wordpress.org/themes/handbook/review/required/#9-files), this should be just one wp.org username, so maybe `wordpressdotorg` is best here. I'm not sure where is the best place to name the contributors. The [guidelines](https://make.wordpress.org/themes/handbook/review/required/#9-files) say 

> Themes can include one single front-facing credit link, which is restricted to the Theme URI or Author URI defined in style.css.

I think we should discuss what's the best way to do this and add that info to the readme file on this repo so we are consistent on all our themes

Reply:
I opened [this PR](https://github.com/WordPress/community-themes/pull/14) for discussion